source/ddbus/meson.build

```diff
@@ -0,0 +1,37 @@
+lib_src = [
+    'attributes.d',
+    'bus.d',
+    'c_lib.d',
+    'conv.d',
+    'exception.d',
+    'package.d',
+    'router.d',
+    'simple.d',
+    'thin.d',
+    'util.d',
+]
+
+ddbus_lib = library(
+    'ddbus',
+    [lib_src],
```

Comment:
Hum? Why do we have an array of arrays? Please just do `lib_src` instead of `[lib_src]`.

Reply:
Ah yes, bad copy&paste there, I do [lib_src, 'test_main.d'] to include a dummy main() file for tests in a project of mine (since GDC doesn't have -main)